packages/amplify-category-notifications/src/channel-email.ts

```diff
@@ -69,7 +69,10 @@ export const enable = async (context:$TSContext, successMessage: string|undefine
   try {
     const data = await context.exeInfo.pinpointClient.updateEmailChannel(params).promise();
     spinner.succeed(successMessage ?? `The ${channelName} channel has been successfully enabled.`);
-    context.exeInfo.serviceMeta.output[channelName] = data.EmailChannelResponse;
+    context.exeInfo.serviceMeta.output[channelName] = {
+      RoleArn: params.EmailChannelRequest.RoleArn,
```

Comment:
The only change in the file is setting the RoleArn from the input parameters, this issue is caused by a bug in the pinpoint CDK not returning the RoleArn in the EmailChannelResponse body as stated in this [documentation](https://docs.aws.amazon.com/pinpoint/latest/apireference/apps-application-id-channels-email.html#apps-application-id-channels-email-response-body-emailchannelresponse-example). Filed a ticket against Pinpoint CDK and this change is meant to be a temporary workaround until the root cause is being fixed

Reply:
Can we do like below if the pinpoint cdk returning undefined or null
```suggestion
      RoleArn: data.EmailChannelResponse ?? params.EmailChannelRequest.RoleArn,
```

Reply:
If the roleArn is an optional field, will it cause problem to send an null or empty string here?